config/webpack.config.js

```diff
@@ -125,6 +130,11 @@ webpackConfig.module.loaders = [{
 
 // ------------------------------------
 // Style Loaders
+//
+// style    - This loader injects <style> tags.
+// css      - Resolves imports and returns css.
+// postcss  - Applies PostCSS processing.
+// sass     - Loads and compiles sass files.
```

Comment:
@chris-bryant I really like what you did here w/ this comment 👍  👍  This is very helpful